### PR TITLE
first stab at Installation in Amazon EKS

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

File: docs-source/docs/modules/deployment/pages/aws-install.adoc [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+----
+
+:!Sectnums:
+== Create a Kubernetes cluster
+
+An Kubernetes cluster can be created from the https://console.aws.amazon.com/iam/home?#/users[Amazon EKS console {tab-icon}, window="tab"] or with the `eksctl` command line tool. For example:
+
+[source,shell script]
+----
+eksctl create cluster --name eks-akka-demo --version 1.17 --region eu-central-1 --nodegroup-name linux-nodes --nodes 3 --nodes-min 1 --nodes-max 4 --with-oidc --managed
```
Comment (patriknw) [inline docs-source/docs/modules/deployment/pages/aws-install.adoc:53]: @chbatey shall `--with-oidc` be included here or not?

**Comment by chbatey** [comment]
not need AFAIK

**Comment by chbatey** [comment]
but will be needed once we have iam service acounts

**Comment by chbatey** [comment]
i'll add instructions for setting up the service account on this

**Comment by patriknw** [comment]
> i'll add instructions for setting up the service account on this

Let's merge this and add more

**Pull request merged**